wp-api.js

```diff
@@ -394,39 +346,32 @@
 	wp.api.models.Page = BaseModel.extend( _.extend(
 		/** @lends Page.prototype  */
 		{
-			idAttribute: 'ID',
+			idAttribute: 'id',
 
 			urlRoot: WP_API_Settings.root + '/pages',
 
 			defaults: {
-				ID: null,
-				title: '',
-				status: 'draft',
-				type: 'page',
-				author: new wp.api.models.User(),
-				content: '',
-				parent: 0,
-				link: '',
+				id: null,
 				date: new Date(),
-				modified: new Date(),
 				date_gmt: new Date(),
+				guid: '',
+				link: '',
+				modified: new Date(),
 				modified_gmt: new Date(),
-				date_tz: 'Etc/UTC',
-				modified_tz: 'Etc/UTC',
-				format: 'standard',
+				password: '',
 				slug: '',
-				guid: '',
+				status: 'draft',
+				type: 'page',
+				title: '',
+				content: '',
+				author: null,
 				excerpt: '',
-				menu_order: 0,
-				comment_status: 'closed',
-				ping_status: 'open',
-				sticky: false,
-				password: '',
-				meta: {
-					links: {}
-				},
 				featured_image: null,
-				terms: []
+				comment_status: 'open',
```

Comment:
@tlovett1 As of WP 4.3 Beta 1 (so...yesterday) `comment_status` and `ping_status` on pages default to `closed`.  Think it is too soon to change the default here?